tools/scripts/update-docker-tags.sh

```diff
@@ -18,12 +18,16 @@ export OLD_TAG=$(cat tools/scripts/environments-target.txt)
 export NEW_TAG="$1"
 
 # get list of images to replace via OLD_TAG in bumpenvs.yaml
-export IMAGES=$(grep -oP "(?<=new: ).*(?=,)" tools/scripts/bumpenvs.yaml | grep -F $OLD_TAG)
+export IMAGES=$(grep -oP "(?<=new: ).*(?=,)" tools/scripts/bumpenvs.yaml | grep -F :$OLD_TAG)
 
 # update tags on dockerhub
 for NAME in $IMAGES; do
     if [ "$2" == "--release" ]; then
-        export NEW_NAME="$(echo $NAME | grep -oP '.*(?=-dev:)'):$NEW_TAG"
+        if [ -z $(echo $NAME | grep -oP '.*(?=-dev:)') ]; then
```

Comment:
I know you aren't changing this, but that regex is kinda weired. Anything, followed by a zero width lookahead assertion of - dev:, followed by anything?  That's just this:
```suggestion
        if [ $NAME != *-dev:* ]; then
```

Reply:
(I suppose noglob should technically be set before running this, just in case there's a file in the current directory)

Reply:
This suggestion is obsoleted by https://github.com/determined-ai/determined/pull/9401#discussion_r1608863910.